src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -553,6 +574,16 @@ void Snippet::schedule_6d(const jit_snippets_call_args& call_args) const {
         });
 }
 
+void Snippet::schedule_6d_per_thread(const std::vector<jit_snippets_call_args>& call_args) const {
```

Comment:
I believe we can avoid additional `per_thread` oveloads. Just create and initilize `jit_snippets_call_args` inside parallel section even if scratcpad is not required. That shouldn't negatively affect the perf.

Reply:
Moved initialization into parallel section